core/templates/core/widget_tag_editor.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n %}
+
+<div data-tags-url="{% url 'api:api-root' %}tags/"
+    {% for k, v in widget.attrs.items %}
+        {{ k }}="{{ v }}"
+    {% endfor %}>
+    {% csrf_token %}
+    <span class="prototype-tag btn badge badge-pill cursor-pointer mr-1" style="display: none;">
+        UNINITIALIZED PROTOTYPE
+        <span class="add-remove-icon pl-1 pr-1">+ or -</span>
+    </span>
+    <div class="current_tags" style="min-height: 2em;">
+        {% for t in widget.value %}
+        <span data-value="{{ t.name }}" data-color="{{ t.color }}" class="tag btn badge badge-pill cursor-pointer mr-1" style="background-color: {{ t.color }};">
+            {{ t.name }}
+            <span class="add-remove-icon pl-1 pr-1">-</span>
+        </span>
+        {% endfor %}
+    </div>
+    <div class="new-tags">
+        <div class="create-tag-inputs input-group">
+            <input class="form-control" type="text" name="" placeholder="{% trans "Tag name" %}">
+            <div class="input-group-append">
+                <button class="btn btn-outline-primary bg-dark btn-add-new-tag" type="button">{% trans "Add" %}</button>
+            </div>
+        </div>
+        <span>{% trans "Recently used:" %}</span>
+        {% for t in widget.tag_suggestions.quick %}
+        <span data-value="{{ t.name }}" data-color="{{ t.color }}" class="tag btn badge badge-pill cursor-pointer mr-1" style="background-color: {{ t.color }};">
+            {{ t.name }}
+            <span class="add-remove-icon pl-1 pr-1">+</span>
+        </span>
+        {% endfor %}
+    </div>
+    <input 
+        type="hidden"
+        name="{{ widget.name }}"
+        value="{% for t in widget.value %}&quot;{{ t.name }}&quot;{% if not forloop.last %},{% endif %}{% endfor %}"
+    >
+
+    <div class="modal fade tag-editor-error-modal">
```

Comment:
Definitely want to do some more custom styling on this but we could always save that for a follow-up issue.